File: js/regex/testEmail.js
var isEmailValid = require("./email.js");

let testEmails = ['summerfang@gmail.com',
				  'summer.fang@gmail.com',
				  '909560719@qq.com', 
				  'asdf asdf@ee.com', 
				  'test@ad',
				  'Abc@aa.dd'];

testEmails.forEach(function(item, index, array){
	if(isEmailValid(item)) {
		console.log(item, "is a valid email address");
	} else {
		console.log(item, "is not a valid email address");
	}
});

File: python/regex/test_email.py
# from email import is_email_valid
from email_validation import is_email_valid

def test_is_email_valid():
    emails = {
        "test@test.com": True,
        "test.test@test.com" : True,
        "test@test": False
    }


    for e in emails:
        if is_email_valid(e) == emails[e]:
            print("Valid")
        else:
            print("invalid")

test_is_email_valid()